analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -220,7 +223,7 @@
     '-[DIUF]',
     '-idirafter',
     '-isystem',
-    '-macros',
+    '-imacros',
```

Comment:
This flag is fixed by #3428 and it is merged. It is very interesting that github doesn't recognise this as a merge conflict but shows this as a change. Can you please rebase your branch to hide this change from this commit? Otherwise this patch looks good to me.

Reply:
Rebased and commit message corrected.